.travis.yml

```diff
@@ -0,0 +1,7 @@
+---
+sudo: false
+language: ruby
+cache: bundler
+rvm:
+  - 2.5.1
```

Comment:
If we're testing for 2.5.x we should test the latest minor release 2.5.5. Even better, we can leverage the Travis build matrix to test our gem against all the [currently maintained ruby versions](https://www.ruby-lang.org/en/downloads/branches/)

```
rvm:
  - 2.4.6
  - 2.5.5
  - 2.6.3
```